packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -481,15 +536,17 @@ class ScrollableBase extends Component {
 
 		this.animateOnFocus = this.props.animate;
 
-		if (!repeat && this.hasFocus()) {
+		if (!repeat && (globalHandler || this.hasFocus())) {
 			const {overscrollEffectOn} = this.props;
 			let
 				overscrollEffectRequired = false,
 				direction = null;
 
 			if (isPageUp(keyCode) || isPageDown(keyCode)) {
 				if (this.props.direction === 'vertical' || this.props.direction === 'both') {
-					Spotlight.setPointerMode(false);
+					if (!globalHandler) {
```

Comment:
If you're able to refactor as suggested above, this guard will likely go away since this method can always disable pointer mode and the global handler will skip this step.

Reply:
Updated!